lib/init/features/functions.js

```diff
@@ -35,6 +35,12 @@ module.exports = function(setup, config) {
     return config.askWriteProjectFile('functions/package.json', {
       name: 'functions',
       description: 'Cloud Functions for Firebase',
+      scripts: {
+        'serve': 'firebase serve --only functions',
```

Comment:
I would favor "firebase experimental:functions:shell" better since it emulates all function types whereas "firebase serve" only emulates HTTPS functions.

Reply:
Interesting. Pasting over docs here for reference:
```
Run Functions Locally
You can run functions locally to test them before deploying to production. To run functions locally, use either:
* The Cloud Functions shell. This tool emulates all types of function triggers, and provides an interactive shell for invoking the functions with test data.
* The Firebase CLI's firebase serve command. This tool emulates HTTPS functions, providing a URL to trigger the local function.
```
I would propose using `firebase experimental:functions:shell` as the default `npm start` script in this case, since it's more fully featured.